plugin.py

```diff
@@ -830,9 +830,9 @@ def karma(self, irc, msg, args, name):
 
         inc, dec = 0, 0
```

Comment:
Good catch, I spent a minute looking at this line to understand the patch and then forgot that this will basically break this PR